src/run.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
jako cargo cult přenáším do všech podobných skriptů tohle

https://github.com/keboola/aws-s3-extractor/blob/851d3bea3ded98b607caaf17ca6c4a181cf14b9a/src/run.php#L3

možná je to nějakej pozůstatek starších PHP a v 7.1 už to není třeba?

Reply:
Myslím, že by to nemělo být potřeba už http://php.net/manual/en/language.errors.php7.php

Reply:
Ale správně bych dole v tom catch bloku měl chytat `Throwable` (protože Error``  nedědí od `Exception`). Díky. 

Mimochodem - na to je taky sniff v Slevomat CS.

Reply:
Ono je to vlastne uplne jedno a dokonce tam ani ten catch na exception/throwable vubec nemusi byt, protoze unhandled exception skonci v php s exitkodem >1 (255) a tim se z toho stane application error

Reply:
Aha, cool. OK.

Reply:
Opraveno.